Devanagari/nepali_romanized.yaml

```diff
@@ -1,128 +1,39 @@
-name: Nepali (Romanized)
+name: Nepali Romanized
 languages: ne-NP
 useZWNJKey: true
 rows:
   - letters: # rowkeys_nepali_romanized1.xml
-    # U+091F: ट DEVANAGARI LETTER TTA
-    # U+093C: ़ DEVANAGARI SIGN NUKTA
-    # U+0920: ठ DEVANAGARI LETTER TTHA
-    - {type: case, normal: ["\u091f", "%", "\u093c"], shifted: ["\u0920"]}
-
-    # U+094C: ौ DEVANAGARI VOWEL SIGN AU
-    # U+0914: औ DEVANAGARI LETTER AU
-    - {type: case, normal: ["\u094c"], shifted: ["\u0914"]}
-
-    # U+0947: े DEVANAGARI VOWEL SIGN E
-    # U+0948: ै DEVANAGARI VOWEL SIGN AI
-    - {type: case, normal: ["\u0947"], shifted: ["\u0948"]}
-
-    # U+0930: र DEVANAGARI LETTER RA
-    # U+0943: ृ DEVANAGARI VOWEL SIGN VOCALIC R
-    - {type: case, normal: ["\u0930"], shifted: ["\u0943"]}
-
-    # U+0924: त DEVANAGARI LETTER TA
-    # U+0925: थ DEVANAGARI LETTER THA
-    - {type: case, normal: ["\u0924"], shifted: ["\u0925"]}
-
-    # U+092F: य DEVANAGARI LETTER YA
-    # U+091E: ञ DEVANAGARI LETTER NYA
-    - {type: case, normal: ["\u092f"], shifted: ["\u091e"]}
-
-    # U+0941: ु DEVANAGARI VOWEL SIGN U
-    # U+0942: ू DEVANAGARI VOWEL SIGN UU
-    - {type: case, normal: ["\u0941"], shifted: ["\u0942"]}
-
-    # U+093F: ि DEVANAGARI VOWEL SIGN I
-    # U+0940: ी DEVANAGARI VOWEL SIGN II
-    - {type: case, normal: ["\u093f"], shifted: ["\u0940"]}
-
-    # U+094B: ो DEVANAGARI VOWEL SIGN O
-    # U+0913: ओ DEVANAGARI LETTER O
-    - {type: case, normal: ["\u094b"], shifted: ["\u0913"]}
-
-    # U+092A: प DEVANAGARI LETTER PA
-    # U+092B: फ DEVANAGARI LETTER PHA
-    - {type: case, normal: ["\u092a"], shifted: ["\u092b"]}
-
-    # U+0907: इ DEVANAGARI LETTER I
-    # U+0908: ई DEVANAGARI LETTER II
-    - {type: case, normal: ["\u0907"], shifted: ["\u0908"]}
+    - {type: case, normal: ["ट", "॰"], shifted: ["ठ"]}  #Q
+    - {type: case, normal: ["ौ"], shifted: ["औ"]}  #W
+    - {type: case, normal: ["े"], shifted: ["ै"]}  #E
+    - {type: case, normal: ["र", "र्‍|ऱ्"], shifted: ["ृ", "ॄ"]}  #R
```

Comment:
See if % is needed here too

Reply:
thank you for pointing this out. I have added % on both locations and on traditional layout